src/components/Home/Home.jsx

```diff
@@ -5,19 +5,56 @@ import { loadNewNotes } from "../../utilities/load-new-note";
 import "./Home.scss";
 import { useNote } from "../../contexts/NoteContext";
 
+import { searchNotes } from "../../utilities/search";
+import { filterNotes } from "../../utilities/filter";
+import { sortNotes } from "../../utilities/sort";
+import { useFilter } from "../../contexts/FilterContext";
```

Comment:
Can create an index.js file in your utilities folder related to the search filter and sort and then export it here in a single line as:
`import {searchNotes,filterNotes,sortNotes} from ../../utilities`